data/benchmarking.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Benchmarking
+ * @difficulty beginner
+ * @tags cli
+ * @run deno bench --allow-net <url>
+ * @resource {https://deno.land/manual@v1.29.1/tools/benchmarker} Deno: Benchmarker tool
+ * @resource {/http-requests} Example: HTTP Requests
+ *
+ * When writing libraries, a very common task that needs to be done is
+ * testing the speed of methods, usually against other libraries. Deno
+ * provides an easy-to-use subcommand for this purpose.
+ */
+
+// The most basic form of deno benchmarking is providing a name and an
+// anonymous function to run.
+Deno.bench("URL parsing", () => {
+  new URL("https://deno.land");
+});
+
+// We are also able to use an async function.
+Deno.bench("Async method", async () => {
+  await 1;
```

Comment:
I think it's better to call actual async APIs here. How about `await crypto.subtle.digest('SHA-256', new Uint8Array([1, 2, 3]))`, for example, which doesn't require any permission